jsk_fetch_robot/jsk_fetch_startup/config/install_udev.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+jsk_fetch_startup=$(builtin cd "`dirname "${BASH_SOURCE[0]}"`"/.. > /dev/null && pwd)
+
+IFS=':' read -r -a prefix_paths <<< "$CMAKE_PREFIX_PATH"
+current_prefix_path="${prefix_paths[0]}"
+
+#
+cd $jsk_fetch_startup/udev_rules
+for file in $(ls ./*.rules); do
+    sudo cp $file /etc/udev/rules.d/
```

Comment:
copy with back up??? 
`cp hoge.py{,.$(date +"%Y%m%d_%H%M%S")}`

Reply:
Thank you so much.
I create backup file.

By the way, how the syntax like `cp hoge.py{,.$(date +"%Y%m%d_%H%M%S")}` is called?
I cannot search this syntax...

```
$ rosrun jsk_fetch_startup install_udev.sh                                                                                                                                                                                                                            
backup /etc/udev/rules.d/60-respeaker.rules to /etc/udev/rules.d/60-respeaker.rules.20210921_185727                                                                                                                                                                   
copied jsk_fetch_startup/udev_rules/60-respeaker.rules to /etc/udev/rules.d/60-respeaker.rules                                                                                                                                                                        
                                                                                                                                                                                                                                                                      
backup /etc/udev/rules.d/70-persistent-net.rules to /etc/udev/rules.d/70-persistent-net.rules.20210921_185727                                                                                                                                                         
copied jsk_fetch_startup/udev_rules/70-persistent-net.rules to /etc/udev/rules.d/70-persistent-net.rules                                                                                                                                                              

backup /etc/udev/rules.d/80-arduino.rules to /etc/udev/rules.d/80-arduino.rules.20210921_185727
copied jsk_fetch_startup/udev_rules/80-arduino.rules to /etc/udev/rules.d/80-arduino.rules

backup /etc/udev/rules.d/99-edgetpu-accelerator.rules to /etc/udev/rules.d/99-edgetpu-accelerator.rules.20210921_185727
copied jsk_fetch_startup/udev_rules/99-edgetpu-accelerator.rules to /etc/udev/rules.d/99-edgetpu-accelerator.rules

backup /etc/udev/rules.d/99-insta360-air.rules to /etc/udev/rules.d/99-insta360-air.rules.20210921_185727
copied jsk_fetch_startup/udev_rules/99-insta360-air.rules to /etc/udev/rules.d/99-insta360-air.rules

backup /etc/udev/rules.d/99-jsk-ps3joy.rules to /etc/udev/rules.d/99-jsk-ps3joy.rules.20210921_185727
copied jsk_fetch_startup/udev_rules/99-jsk-ps3joy.rules to /etc/udev/rules.d/99-jsk-ps3joy.rules
```